### PR TITLE
add simple completion support for zsh

### LINKED ISSUE #435: crictl completion for zsh
#213 states that installing the completion using `source <(crictl completion)` works for both bash and zsh. Unfortunately this is only true if `bashcompinit` has been loaded in your zshrc. Depending on your zsh setup this might or might not be the case when you source the output of the completion sub-command.

As far as i undertstand this could be fixed by adding the contents of this [file](https://github.com/urfave/cli/blob/master/autocomplete/zsh_autocomplete) to [cmd/crictl/completion.go](../blob/master/cmd/crictl/completion.go) and add a parameter to the completion sub-command that selects between bash or zsh.

### PR DESCRIPTION
**Pull request by cpointner**
fixes #435.

### CODE REVIEW

File: cmd/crictl/completion.go [hunk 1]
```diff
@@ -34,33 +34,95 @@ var bashCompletionTemplate = `_cli_bash_autocomplete() {
...
-		fmt.Fprintln(c.App.Writer, fmt.Sprintf(bashCompletionTemplate, strings.Join(subcommands, "\n")))
-		return nil
+		shell := c.Args().First()
+		switch shell {
+		case "bash":
+			return bashCompletion(c)
+		case "zsh":
+			return zshCompletion(c)
+		default:
+			return fmt.Errorf("unknown shell: %q", shell)
```
Comment (feiskyer) [inline cmd/crictl/completion.go]: could we log only bash and zsh supported?

Comment (cpointner) [reply cmd/crictl/completion.go]: yeah you are right. Just pushed the change, do you want me to squash it together?

Comment (feiskyer) [reply cmd/crictl/completion.go]: thanks

**Comment by feiskyer** [comment]
close-reopen for re-trigger Travis CI

**Pull request merged**